modules/Bio/EnsEMBL/Analysis/Hive/Config/TranscriptSelection.pm

```diff
@@ -0,0 +1,910 @@
+    'release_number'            => '' || $self->o('ensembl_release'),
+    'species_name'              => '',                                                          # e.g. mus_musculus
+    'production_name'           => '',                                                          # usually the same as species name but currently needs to be a unique entry for the production db, used in all core-like db names
+    'uniprot_set'               => '',                                                          # e.g. mammals_basic, check UniProtCladeDownloadStatic.pm module in hive config dir for suitable set,
+    'output_path'               => '',                                                          # Lustre output dir. This will be the primary dir to house the assembly info and various things from analyses
+    'assembly_name'             => '',                                                          # Name (as it appears in the assembly report file)
+    'assembly_accession'        => '',                                                          # Versioned GCA assembly accession, e.g. GCA_001857705.1
+    'assembly_refseq_accession' => '',                                                          # Versioned GCF accession, e.g. GCF_001857705.1
+    'registry_file'             => '' || catfile( $self->o('output_path'), "Databases.pm" ),    # Path to databse registry for LastaZ and Production sync
+    'use_genome_flatfile'       => '1',                                                         # This will read sequence where possible from a dumped flatfile instead of the core db
+    'skip_projection'           => '0',                                                         # Will skip projection process if 1
+    'skip_rnaseq'               => '0',                                                         # Will skip rnaseq analyses if 1
+    'skip_cleaning'             => '0',                                                         # Will skip the cleaning phase, will keep more genes/transcripts but some lower quality models may be kept
+
+########################
+# Pipe and ref db info
+########################
+
+    'projection_source_db_name'         => '',                                                  # This is generally a pre-existing db, like the current human/mouse core for example
+    'projection_source_db_server'       => 'mysql-ens-mirror-1',
+    'projection_source_db_port'         => '4240',
+    'projection_source_production_name' => '',
+
+    'cdna_db_server' => $self->o('databases_server'),
+    'cdna_db_port'   => $self->o('databases_port'),
+
+    'genblast_db_server' => $self->o('databases_server'),
+    'genblast_db_port'   => $self->o('databases_port'),
+
+    'projection_db_server' => $self->o('databases_server'),
+    'projection_db_port'   => $self->o('databases_port'),
```

Comment:
Could you please change projection_db to selected_projection_db? This is to make the config file more explicit